prisma-fmt/tests/code_actions/scenarios/relation_mode_prisma_missing_index/result.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Are we testing anything here?

Reply:
Placeholder until diagnostic is finalised per #3429